packages/block-library/src/code/theme.scss

```diff
@@ -1,9 +1,11 @@
 .wp-block-code {
 	border: 1px solid #ccc;
 	border-radius: 4px;
+	font-family: Menlo, Consolas, monaco, monospace;
+	padding: 0.8em 1em;
 }
 
+/* Ensures code markup inherits the font-family set on pre. */
 .wp-block-code > code {
-	font-family: Menlo, Consolas, monaco, monospace;
-	padding: 0.8em 1em;
+	font-family: inherit;
```

Comment:
I think 'font-family: inherit;' should be in packages/block-library/src/code/style.scss. Even if theme.scss is not enqueued we want font families set on global styles to work well.

Reply:
In that light, the rule could be _moved_ to style.scss, since that always gets loaded even if you also load theme.scss.

Reply:
I just moved the code `font-family` styling to style.css. That makes more sense and works as expected. Good catch 🙌